docs/PREREQUISITES.md

```diff
@@ -0,0 +1,61 @@
+# Prerequisites #
+## Requirements ##
+
+In order to deploy Atlassian’s Data Center products, the following is required:
```

Comment:
I think we should also include the expectation of a certain understanding of Kubernetes and Helm concepts.